apps/nextjs/src/components/AppComponents/Chat/chat-history.stories.tsx

```diff
@@ -15,11 +16,21 @@ export default meta;
 type Story = StoryObj<typeof ChatHistory>;
 
 export const Default: Story = {
+  render: () => (
+    <Dialog>
+      <ChatHistory />
+    </Dialog>
+  ),
   args: {
     userId: "user123",
   },
 };
 
 export const WithoutUserId: Story = {
+  render: () => (
+    <Dialog>
+      <ChatHistory />
+    </Dialog>
+  ),
```

Comment:
I've handled this with a decorator on the component in [my storybook PR](https://github.com/oaknational/oak-ai-lesson-assistant/pull/344/files#diff-7e738b622951adac91695c4d030749492727fb6e9d1f36fe3340d96ef05f351f)

There's a chance we'll get two providers if we merge both of these

Reply:
Okay thanks for the warning on that!